core/src/actions/schedules.ts

```diff
@@ -210,13 +210,17 @@ export class ScheduleEdit extends AuthenticatedAction {
 
   async runWithinTransaction({ params }: { params: ParamsFrom<ScheduleEdit> }) {
     const schedule = await Schedule.findById(params.id);
+
+    const recurringFrequency =
+      params.recurring && params.recurringFrequency
+        ? params.recurringFrequency
+        : 0;
+
     // these timing options are validated separately, and should be set first
-    if (params.recurringFrequency || params.recurring) {
```

Comment:
Here's the problem: Only updates recurring if it's `true`.

Reply:
@evantahler Another question that came to mind: If we call this action and we don't set a recurring/recurring frequency, should it update the current recurring frequency or should I really check if these values are in the params before making updates?